lib/logger/lib/logger.ex

```diff
@@ -555,6 +555,7 @@ defmodule Logger do
     :level,
     :utc_log,
     :discard_threshold,
+    :discard_threshold_periodic_check,
```

Comment:
Should this option be documented at the module documentation?

Reply:
I don't want to document it for now, because I don't think it actually has any use. Especially since it is v1.8 only. Our default is likely good enough in all cases.

Reply:
Oh! Ok. I mentioned it because it’s documented in `master`

https://github.com/elixir-lang/elixir/blob/dc154f5db7a54173decd00f7971e9d8d41aa59af/lib/logger/lib/logger.ex#L152